pkgs/development/compilers/llvm/8/clang/clang_qtcreator_support.patch

```diff
@@ -0,0 +1,334 @@
+diff --git a/tools/driver/CMakeLists.txt b/tools/driver/CMakeLists.txt
```

Comment:
Is there an upstream version of this patch? If possible it would be preferred to use ```fetchpatch``` to avoid checking this in.

Reply:
Unfortunately no, qt maintain separate version of clang in their own git, and afaik deliver it through their installer package (binary .run file). It's is possible, however, to fetch patch directly from their tracker by [link](https://code.qt.io/cgit/clang/clang.git/patch/?id=7f349701d3ea0c47be3a43e265699dddd3fd55cf). Please let me know if i should do it this way.

Reply:
Yeah that works

Reply:
I tried using fetchpatch, but that fails (that diff is against older version of clang). I suggest leaving patch file as it is now.